Formula/s/sitefetch.rb

```diff
@@ -0,0 +1,21 @@
+class Sitefetch < Formula
+  desc "Fetch an entire site and save it as a text file"
+  homepage "https://github.com/egoist/sitefetch"
+  url "https://registry.npmjs.org/sitefetch/-/sitefetch-0.0.17.tgz"
+  sha256 "eb65e7d0179e5c06f70a3071865cf8199827428adb64da19de12afaf682c0ddd"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *std_npm_args
+    bin.install_symlink libexec/"bin/sitefetch"
```

Comment:
no reason to hardcode this binary (which will get out of sync if upstream changes or has multiple binaries)

```suggestion
    bin.install_symlink libexec.glob("bin/*")
```

fyi I'm also fixing some other cases in #208225

Reply:
but there is just one binary in the libexec/bin? why do we prefer glob approach?

Reply:
It's more maintainable for us since it doesn't required any changes on our side no matter what changes upstream (for example, if they rename the binary, add more, remove some, etc)